oresat_dxwifi/services/camera.py

```diff
@@ -0,0 +1,95 @@
+"""Oresat Live Camera Service"""
+
+from olaf import Service, logger
+from enum import IntEnum
+from ..camera.camera import CameraInterface
+
+
+class State(IntEnum):
+    OFF = 0
+    BOOT = 1
+    STANDBY = 2
+    FILMING = 3
+    ERROR = 0xFF
+
+
+STATE_TRANSISTIONS = {
+    State.OFF: [State.BOOT],
+    State.BOOT: [State.STANDBY],
+    State.STANDBY: [State.FILMING],
+    State.FILMING: [State.STANDBY, State.ERROR],
+    State.ERROR: [],
+}
+
+
+class CameraService(Service):
+    def __init__(self):
+        super().__init__()
+        self.state = State.BOOT
+
+        self.spv = 3  # Should evenly divide into duration
+        self.duration = 6
+        self.IMAGE_OUPUT_DIRECTORY = "/oresat-live-output/frames"  # Make sure directory actually exists
```

Comment:
Don't we create it if it doesn't?

Reply:
No we just don't capture. The way the capture stuff is set up down the line just expects it I guess. Still something to consider changing since we set the video configurations here now. A lot of this stuff still needs cleanup because it tried to maintain the integrity of oresat-live-software/camera cmd functionality.

Reply:
What about in `create_videos()` where it calls `self._fresh_dir(self.image_output_directory)`?

Reply:
Oh I misunderstood what you were referring to. That doesn't create the directories, it just clears them. It needs them to exist. Its internal logic uses a receiving an error to determine whether to remove a directory or file. I think that's why I didn't try to rework it.

Reply:
It's a remaining component from oresat-live-software/camera

Reply:
```
    def create_videos(self) -> None:
            self._fresh_dir(self.image_output_directory)

    def _fresh_dir(path) -> None:
        if os.path.exists(path):
            ...
        else:
            # Make
            os.mkdir(path)
```